rslib/src/pb.rs

```diff
@@ -29,3 +29,4 @@ protobuf!(search, "search");
 protobuf!(stats, "stats");
 protobuf!(sync, "sync");
 protobuf!(tags, "tags");
+protobuf!(themes, "themes");
```

Comment:
I'd probably just put this in config.proto, as that's where the rest of the preferences stuff lies.